File: ELearningBackEnd/src/main/java/com/example/elearningbackend/order/RefundReq.java
package com.example.elearningbackend.order;

import lombok.Getter;
import lombok.Setter;

@Getter
@Setter
public class RefundReq {

    private long orderDetailId;

    private long courseId;

    private Long discountId;

    private String reason;

    private double totalAmount;

    private double discountAmount;

    private double finalAmount;
}

File: ELearningFrontEnd/src/app/dtos/order/refund.req.ts
export class RefundReq{

    public courseId: number;
    public discountId: number| null;
    public reason: string;
    public totalAmount: number;
    public discountAmount: number;
    public finalAmount: number;
    public orderDetailId: number;
    constructor(data: any) {

        this.courseId = data.courseId;
        this.discountId = data.discountId;
        this.reason = data.reason;
        this.totalAmount = data.totalAmount;
        this.discountAmount = data.discountAmount;
        this.finalAmount = data.finalAmount;
        this.orderDetailId = data.orderDetailId;
    }
}